tests/parser/tests_IStreamQuadIterator.cpp

```diff
@@ -308,6 +308,20 @@ TEST_SUITE("IStreamQuadIterator") {
         CHECK_EQ(qit, std::default_sentinel);
     }
 
+    TEST_CASE("no bnode parsing") {
+        constexpr char const *triples = "<a> <b> _:bnode .\n";
+
+        std::istringstream iss{triples};
+        IStreamQuadIterator qit{iss, ParsingFlag::NoParseBlankNode};
+
+        CHECK_NE(qit, std::default_sentinel);
+        CHECK(!qit->has_value());
+        std::cout << qit->error() << std::endl;
```

Comment:
check that the message contains something from `"Encountered blank node while parsing"`.